spec/system/provider_interface/update_interview_spec.rb

```diff
@@ -0,0 +1,91 @@
+require 'rails_helper'
+
+RSpec.describe 'A provider user changes interview details' do
```

Comment:
Consider merging these steps in the [manage interview spec](https://github.com/DFE-Digital/apply-for-teacher-training/blob/master/spec/system/provider_interface/provider_manages_application_interviews_spec.rb#L36) after the creation and before the cancellation steps as we will then be testing all aspects of this feature via the interface and ensure everything works well together.

Reply:
:+1: https://github.com/DFE-Digital/apply-for-teacher-training/pull/3993/commits/e09a5048bd9652981dd9cec3ef3c39162515339b

Reply:
Can we remove this spec now?